# Service that returns the name of the current detector
---
string current_detector         # Name of current detector
